docs/articles/faq/README.md

```diff
@@ -0,0 +1,253 @@
+  * [Which browsers does TestCafe support? What are the exact supported versions?](#which-browsers-does-testcafe-support-what-are-the-exact-supported-versions)
+  * [Can I use third-party modules in tests?](#can-i-use-third-party-modules-in-tests)
+  * [How do I work with configuration files and environment variables?](#how-do-i-work-with-configuration-files-and-environment-variables)
+* [Troubleshooting](#troubleshooting)
+  * [I have installed TestCafe but I cannot run it. What's your advice?](#i-have-installed-testcafe-but-i-cannot-run-it-whats-your-advice)
+  * [When I run a TestCafe test, I get an unexpected error. Do you know any possible reasons?](#when-i-run-a-testcafe-test-i-get-an-unexpected-error-do-you-know-any-possible-reasons)
+  * [I have installed TestCafe plugins but they do not work. What have I done wrong?](#i-have-installed-testcafe-plugins-but-they-do-not-work-what-have-i-done-wrong)
+  * [My test fails because TestCafe could not find the required webpage element. What could go wrong?](#my-test-fails-because-testcafe-could-not-find-the-required-webpage-element-what-could-go-wrong)
+
+## General Questions
+
+### I've heard that TestCafe doesn't use Selenium under the hood. How does it actually operate?
+
+Unlike most testing solutions, TestCafe isn't built on top of Selenium.
+This allows us to implement the unique features you won't find in Selenium-based tools
+(e.g., testing on mobile devices, user roles, automatic waiting, etc).
+
+You may wonder how TestCafe manages to work without the WebDriver.
+Under the hood, TestCafe uses a [URL-rewriting proxy](https://github.com/DevExpress/testcafe-hammerhead).
+This proxy injects the driver script that emulates user actions right into the tested page.
+
+You can read more about our motivations on our [forum](https://testcafe-discuss.devexpress.com/t/why-not-use-selenium/47).
+Don't hesitate to ask for more details.
+
+### I've noticed there are two versions of TestCafe: a [paid version](https://testcafe.devexpress.com) and an [open-source version](https://devexpress.github.io/testcafe). Why have two versions and what's the difference between them?
+
+Historically, TestCafe first appeared as a paid standalone tool. Beside the test runner,
+it featured the [Control Panel](https://testcafe.devexpress.com/Documentation/Using_TestCafe/Control_Panel/),
+a visual interface for managing your tests (creating, modifying, running), and an innovative
+[Visual Test Recorder](https://testcafe.devexpress.com/Documentation/Using_TestCafe/Visual_Test_Recorder/)
+that allowed you to record tests by pointing and clicking through the test scenario in the browser.
+
+This version was actively developed until 2015 when we decided to release its core as an open-source project.
+The last major release of the paid version happened in the summer 2015, which was v15.1.
```

Comment:
In 2015 we decided to release its core as an open-source project. The last major release of the paid version (v15.1) happened in the summer 2015.